en/engine/event/event-input.md

```diff
@@ -0,0 +1,100 @@
+# Global System Events
+
+In this section, we will introduce the global system events of Cocos Creator.
+
+Global system events are irrelevant with the node hierarchy, so they are dispatched globally by `cc.systemEvent`, currently supported:
+
+- Mouse
+- Touch
+- Keyboard
+- DeviceMotion
+
+The mouse event and the touch event are similar to the node system events, except that the area of action is different. The following is a description of these events.
+
+## The difference between node events and global mouse and touch events
+
+Before we begin this section, I hope you'll read up on [Auto fit for multi-resolution](../../ui-system/components/engine/multi-resolution.md#Design-resolution-and-screen-resolution) and  understand the screen area and UI display area. When listening for global mouse/touch events, the acquired contacts are calculated based on the bottom left corner of the screen area (device display resolution). The contacts fetched by the UI node listener are not the same as the contacts fetched by the global event, which are converted to the points calculated in the lower left corner of the adapted UI viewable area. Global touch points are better suited for manipulating the behavior of 3D nodes by tapping directly on the screen, without having to add UI nodes to the scene to listen for mouse/touch events.
```

Comment:
Before we begin this section, I hope you'll read up on [Auto fit for multi-resolution](../../ui-system/components/engine/multi-resolution.md#Design-resolution-and-screen-resolution) and  understand the screen area and UI display area.
-->
> **Note**: Before begining this section, read up on [Auto fit for multi-resolution](../../ui-system/components/engine/multi-resolution.md#Design-resolution-and-screen-resolution) and  understand the screen area and UI display area.